README.md

```diff
@@ -8,10 +8,18 @@
   * Set the `MSISDN` environment variable 
 
 ### Run
-Run via **command line** or an **IDE** like IntelliJ or Visual Studio Code to launch the example *Main.kt files. 
+Run via an **IDE** like IntelliJ or Visual Studio Code to launch the example *Main.kt files, or via **command line**.
+
+In any case you will have to supply the needed parameters as environment variables.
 
 #### Via command line
-**Example**
+
+##### Voicemail
 ```shell script
 $ WGTWO_CLIENT_ID=AbC... WGTWO_CLIENT_SECRET=dEf... MSISDN=4799900111 ./mvnw exec:java -Dexec.mainClass="com.wgtwo.example.voicemail.VoicemailMainKt"
 ```
+
+##### Send SMS
+```shell script
+$ WGTWO_CLIENT_ID=AbC... WGTWO_CLIENT_SECRET=dEf... FROM=4799900111 TO=4799900111 CONTENT="Hi" ./mvnw exec:java -Dexec.mainClass="com.wgtwo.example.sendsms.SendSmsMainKt"
```

Comment:
Maybe we also should include some simple scripts to simplify this?

So include a `run/sendsms` script with something like this?
```
#!/usr/bin/env sh
./mvnw exec:java -Dexec.mainClass="com.wgtwo.example.sendsms.SendSmsMainKt" -Dexec.args="$@"
```